src/metabase/pulse/render/style.clj

```diff
@@ -95,7 +91,7 @@
   []
   (merge
    (font-style)
-   {:font-size       :14px
+   {:font-size       :18px
```

Comment:
The design header font size is 32px with 22px table font size that's 32/22 = 1.45 times the table font size

The current table font size is 12px, so that came to 12*1.45 = 17.4, I round this number up so I got 14px.